std/algorithm/internal.d

```diff
@@ -74,5 +74,4 @@ version(unittest)
     }
 }
 
-package T* addressOf(T)(ref T val) { return &val; }
-
+package(std) T* addressOf(T)(ref T val) { return &val; }
```

Comment:
Shouldn't this be moved somewhere outside of `std.algorithm`? Especially since it's imported from other places, like `std.range` below.